src/IROperator.h

```diff
@@ -397,7 +414,10 @@ Expr &operator*=(Expr &a, Expr b);
 /** Return the ratio of two expressions, doing any necessary type
  * coercion using \ref Internal::match_types. Note that signed integer
  * division in Halide rounds towards minus infinity, unlike C, which
- * rounds towards zero. */
+ * rounds towards zero. Division by zero returns zero. For types where
+ * overflow is defined behavior, division of the largest negative
+ * signed integer by -1 returns the larged negative signed integer for
+ * the type (i.e. it wraps). */
 Expr operator/(Expr a, Expr b);
```

Comment:
I would like to see this sort of documentation added to the tutorial since it's such an unusual behavior.